docs/cli-usage.md

```diff
@@ -89,7 +89,7 @@ djinni \
 | `--jni-include-prefix <prefix>` | The prefix for `#includes` of JNI header files from JNI C++ files. |
 | `--jni-include-cpp-prefix <prefix>` | The prefix for `#includes` of the main header files from JNI C++ files. |
 | `--jni-namespace ...` | The namespace name to use for generated JNI C++ classes. |
-| `--jni-generate-main <true/false>` | Generate a source file (`djinni_jni_main.cpp`) that includes the default `JNI_OnLoad` & `JNI_OnUnload` implementation from the support library. (default: `true`) |
+| `--jni-generate-main <true/false>` | Generate a source file (`djinni_jni_main.cpp`) that includes the default `JNI_OnLoad` & `JNI_OnUnload` implementation from the support library. (default: `false`) |
```

Comment:
Hi, why it need be true? Is not true that everybody want generate main file. It need be true? We need put true as default what is related to files generation, not for support library files.

Reply:
It is true because it is a sane default that does what most people need. And it's also what djinni always did, so we will not break compatibility.

I know, Paulo, you have an 'special' use case where you want to generated that part your own.
I would be very happy if that use case would land in the documentation.

But most users have just 1 library they generate and link, and for those this default is what they want and need.

Reply:
I respect your opinion, but don't agree. Djinni generate glue code files, not support files for default. In my opinion, all new options need be false or disabled. But do what you want, i back it to false.

Reply:
@paulo-coutinho if you still have concerns regarding the current default I think we should discuss this in a separate PR/issue. :)
This PR should be focused on the WAMS feature only.